### ISSUE TITLE
Research default theme

### ISSUE THREAD

**Issue by dimitraz** [slant-website/breakpoint.css#2]
**Description**
Decide on 
- 3 main font choices 
- default colour palette

**Comment by kingsleyzissou** [comment]
Avenir

**Comment by dimitraz** [comment]
Sans - Helvetica
Mono - Roboto Mono

**Comment by kingsleyzissou** [comment]
Yeah I think we should have a .mono/.code class and I like robot mono
I'm not too sure about Helvetica though.

We also need a serif font I reckon

**Comment by kingsleyzissou** [comment]
With regards to the colours - I think we should have 3 colours like you say.

But maybe some basic UI/UX colours for form feedback in inputs, notification, badges, etc. Thoughts?

**Comment by dimitraz** [comment]
@kingsleyzissou yep agree

**Comment by piccolazucca** [comment]
So the solution to our framework resulting outdated once any trend has passed is to offer multiple fonts, and have a class with the font name or type (.helvetica, or .serif, .mono)

Fonts I've picked:

Titles:
- [Krona One](https://fonts.google.com/specimen/Krona+One) (much hip, many cool)
- [Montserrat](https://fonts.google.com/specimen/Montserrat)
Body text:
- Helvetica (sans-serif)
- Times New Roman (serif) -> tbc
Other:
- [Space Mono](https://fonts.google.com/specimen/Space+Mono) (mono)

Although I've set categories (for titles and such) of course you will be able to use them interchangeably.

**Comment by dimitraz** [comment]
idk about using font classes if you wanted to customise the whole framework it seems redundant and not scaleable. I think a better solution would find a way to override the default theme with a custom theme

**Comment by milenazuccarelli** [comment]
Circling back to this discussion, I'd like to pick out more _timeless_ fonts. I'll source a few more and make a PR in the coming days
